tests/ping.go

```diff
@@ -0,0 +1,81 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2020 Red Hat, Inc.
+ *
+ */
+
+package tests
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	expect "github.com/google/goexpect"
+	"google.golang.org/grpc/codes"
+	"k8s.io/utils/net"
+
+	v1 "kubevirt.io/client-go/api/v1"
+)
+
+const promptExpression = "\\$ |\\# "
+const termNLExpression = "\r\n"
+
+var (
+	shellSuccess = regexp.MustCompile("\n0" + termNLExpression + ".*" + promptExpression)
```

Comment:
There may be abug here, expessions with "\\# " only may be matched. `termNLExpression` should be surrounded with parenthesis.

Reply:
same for promptExpression isnt it ?

Reply:
Yes, `promptExpression`. Edited the comment.
Was fixed there as well of course. That's how I found the issue.

Reply:
Done